src/Honeycomb/HoneycombService.cs

```diff
@@ -73,7 +73,7 @@ public async Task SendSingleAsync(HoneycombEvent ev)
             var content = JsonConvert.SerializeObject(ev.Data);
             message.Content = new StringContent(content, Encoding.UTF8, "application/json");
             message.Method = HttpMethod.Post;
-            message.RequestUri = new Uri($"https://api.honeycomb.io/1/events/{WebUtility.UrlEncode(ev.DataSetName)}");
+            message.RequestUri = new Uri($"https://{_settings.Value.ApiHost}/1/events/{WebUtility.UrlEncode(ev.DataSetName)}");

```

Comment:
I had considered having the setting include the scheme so that we could support using both http and https endpoints.  Thoughts?

Reply:
Hmm, yes that would probably be helpful and looks to be consistent with our other integrations. Sorry for jumping the gun on merging. Would you like to create a new PR for that addition?

Reply:
Done. https://github.com/honeycombio/libhoney-dotnet/pull/21